lib/erlex.ex

```diff
@@ -98,66 +98,47 @@ defmodule Erlex do
           function :: String.t()
         ) :: String.t()
   def pretty_print_contract(contract, module, function) do
-    multiple_heads? =
+    [head | tail] =
       contract
       |> to_string()
-      |> String.contains?(";")
+      |> String.split(";")
 
-    # TODO: This is kind of janky but I've only seen this once and am
-    # not sure how to make it happen generally.
-    if multiple_heads? do
-      [head | tail] =
-        contract
-        |> to_string()
-        |> String.split(";")
-
-      head =
+    head =
+      if tail != [] do
```

Comment:
Can you flip these clauses so this is a positive conditional?

Reply:
Actually case is probably more expressive here.

Reply:
Sure, will do!

Reply:
Done with 1f8c75e.